committee-steering/sig-governance.md

```diff
@@ -0,0 +1,533 @@
+- A proposal with [lazy consensus][lazy-consensus] within the SIG.
+- SIG architecture picking technical leads for the SIG
+
+### Member role
+
+Canonical list of members: kubernetes-sig-~~foo~~-members@googlegroups.com
+
+### Responsibilities towards members the SIG
+
+- Acting as role model to other members within the SIG
+  - Maintaining a respectful and welcoming culture is the responsibility
+    of all members of the SIG.  Treat other members with kindness.
+  - Foster an inclusive culture
+- Ensuring continued area health through proactively
+  - Triaging issues
+  - Reviewing PRs
+  - Participation in monitoring of test health
+  - Fixing tests
+  - Contributing features
+  - Refactoring and simplifying codebase
+  - Participating in design discussions and providing feedback
+- Taking technical ownership of areas governed by the SIG
+  - Required: Ensuring health of code authored
+  - Optional: Ensuring health of code authored by other members
+  - Health includes
+    - contributions adhere to criteria defined by style and contribution guides
+    - code correctness is tested through automation
+    - tests remain passing
+    - issues are resolved and patched if needed
+    - code remains simple and approachable to others
+      - self documenting code through function and variable naming
+      - explicit documentation
+  - Making technical decisions with other members
+- Responding to issues facing the SIG
+  - Fixing critical bugs
+
+### Externally facing responsibilities
+
+- Acting as ambassadors to the SIG through
+  - Optional: Responding to messages on communication channels
+  - Optional: Participating in events such as meetups
+
+#### Selecting members
+
+Members should show a commitment to the SIG before becoming members.
+This maybe a verbal commitment - e.g. I plan to work in this area
+full time for the next 3 months - or something demonstrated over time
+through contributions in PRs, issue triage, test triage, operations,
+etc.
+
+Members are expected to be responsive to issues for sub areas
+they own within the SIG.  Members are expected to help other
+members in the SIG (e.g. transfer knowledge, help debug, assist with
+operational tasks, answer questions, etc).
+
+Membership is provisional until the member has shown sustained
+contributions for 3 months.  Provisional members may provide
+feedback in decision making, but are not deciders in the decision
+itself.
+
+## Processes and tools
+
+The following section outlines various processes used by the SIG.
+
+### Decision making within the SIG
+
+It is critical to a SIG that the decision making processes is healthy.
+A good decision making process has the following properties:
+
+- Decision making process is unambiguously understood by everyone involved
+- Decisions are resolved instead of languishing and defaulting to no decision
+- Decisions are a commitment and won't waffle after having been made
+
+#### Who, What, Where, When, How
```

Comment:
It may be self-evident, but I think it may help to note that a proposal is best offered early in a release cycle to ensure adequate time for review, ideally the item aligns with the sig roadmap and/or identified release features, and sigs may constrain the rate of change in any release in order to ensure high quality.

Reply:
Done.  PTAL